saleor/graphql/checkout/dataloaders.py

```diff
@@ -245,14 +254,11 @@ def with_checkout_info(results):
                             delivery_method = collection_points_map.get(
                                 checkout.collection_point_id
                             )
-                        shipping_address = (
-                            address_map.get(checkout.shipping_address_id),
-                        )
+                        shipping_address = address_map.get(checkout.shipping_address_id)
```

Comment:
This was previously a list and now is an instance - was it a bug until now?

Reply:
all the methods that use `shipping_address` below seem to be expecting an `Address` instance, not a one element tuple. I think before it was just a typo or maybe a linter messed up somehow